flink-table/flink-table-api-scala-bridge/src/test/scala/org/apache/flink/table/api/bridge/scala/internal/StreamTableEnvironmentImplTest.scala

```diff
@@ -48,8 +48,9 @@ class StreamTableEnvironmentImplTest {
     val table = tEnv.fromDataStream(elements)
     tEnv.toDataStream(table)
 
-    assertThat(tEnv.getConfig.getMinIdleStateRetentionTime).isEqualTo(retention.toMillis)
-    assertThat(tEnv.getConfig.getMaxIdleStateRetentionTime).isEqualTo(retention.toMillis * 3 / 2)
+    assertThat(tEnv.getConfig.getIdleStateRetention.toMillis).isEqualTo(retention.toMillis)
+    assertThat(tEnv.getConfig.getIdleStateRetention.toMillis * 3 / 2)
```

Comment:
nit: It seems that this assertion can be removed:
```
assertThat(tEnv.getConfig.getIdleStateRetention.toMillis * 3 / 2)...
```

Reply:
done :)